test.js

```diff
@@ -355,3 +355,36 @@ test('doesn\'t write to disk upon instanciation if and only if the store didn\'t
 	exists = fs.existsSync(conf.path);
 	t.is(exists, true);
 });
+
+test('schema - invalid set', t => {
+	const schema = {
+		$schema: 'http://json-schema.org/draft-07/schema#',
+		type: 'object',
+		properties: {
```

Comment:
We should set these top-level meta things for the user. The user should only specify their schema as:

```js
{
	foo: {
		type: 'string'
	}
}
```